src/pages/workspaces/workspace/analysis/utils/app-utils.ts

```diff
@@ -90,3 +90,22 @@ export const getAppStatusForDisplay = (status: LeoAppStatus): DisplayAppStatus =
     ['provisioning', () => 'Creating'],
     [Utils.DEFAULT, () => _.capitalize(status)]
   );
+
+export const getEnvMessageBasedOnStatus = (app: App | undefined): string => {
```

Comment:
this looks like an easy win to get unit test coverage here.  80% is the goal on new/modified code.